openmetadata-ui/src/main/resources/ui/src/components/nav-bar/NavBar.tsx

```diff
@@ -66,7 +88,112 @@ const NavBar = ({
     debouncedOnChange,
   ]);
 
-  const currentUser = AppState.getCurrentUserDetails();
+  const handleBellClick = (visible: boolean) => {
+    if (visible) {
+      switch (activeTab) {
+        case 'Task':
+          hasTaskNotification &&
+            setTimeout(() => {
+              setHasTaskNotification(false);
+            }, 5000);
+
+          break;
+
+        case 'Conversation':
+          hasMentionNotification &&
+            setTimeout(() => {
+              setHasMentionNotification(false);
+            }, 5000);
+
+          break;
+      }
+    }
+  };
+
+  const handleTaskNotificationRead = () => {
+    setHasTaskNotification(false);
```

Comment:
We are doing the same thing in line number `97` and `105`, can we move these functions up and use the same there as well?